docs/usage-limits.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: usage-limits
+append_help_link: true
+hide_title: true
+description: >-
+  Frequently asked questions about Semgrep, comparisons to similar tools,
+  rule licensing, technical support, and more.
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier.
+
+If the number of contributors has exceeded the usage limit, you will need to purchase licensing for the 11th contributor onward. 
```

Comment:
Use simple present tense in documentation as much as possible.

Optional: Avoid repetitive overuse of "usage limit". We can use the limit of 10 contributors maybe in some cases?